pyanaconda/network.py

```diff
@@ -214,6 +218,22 @@ def iface_for_host_ip(host_ip):
     return route_info[route_info.index("dev") + 1]
 
 
+def copy_resolv_conf_to_root(root="/"):
+    """Copy resolv.conf to a system root."""
+    src = "/etc/resolv.conf"
+    dst = os.path.join(root, src.lstrip('/'))
+    if not os.path.isfile(src):
+        log.debug("%s does not exist", src)
```

Comment:
Maybe the debug messages could be more self explanatory? I'm not sure if I would understand why this information should be interesting to me from the message.

Reply:
Agreed, I think it should mention something about resolv conf in the error message. :)

Reply:
I didn't want to modify a revert. Also, the message says "/etc/resolv.conf does not exist" in the log in context of the "Copy resolv.conf to sysroot" task. Which is quite clear I think.

Reply:
OK, sorry I missed it is a revert - my baf